moveit_ros/planning/planning_request_adapter_plugins/res/default_plan_request_adapter_params.yaml

```diff
@@ -0,0 +1,44 @@
+default_plan_request_adapter_parameters:
```

Comment:
It's not usually standard to call these files `default_` ... the file defines the parameters, but also happens to provide default values.

Maybe just remove the `default_` from the name?

Reply:
The default is referring to the group of plugins for which this file provides parameters. Those are called `default_plan_request_adapters` but I get why this might be confusing. Do you think we should rename it nonetheless?

Reply:
Ah! I see, then it's fine to leave it.